jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -266,13 +266,12 @@ YUI.add('deployment-flow', function() {
      @method _handleLogin
     */
     _handleLogin: function() {
-      const controllerAPI = this.props.controllerAPI;
-      controllerAPI.connect();
-      const handler = controllerAPI.after('facadesChange', e => {
-        if (e.newVal && controllerAPI.get('facades')) {
```

Comment:
This was required because we were only connecting to the controller after logging in. If we're going to open the controller connection right away then it "shouldn't" be a problem